pynecone/state.py

```diff
@@ -357,6 +357,18 @@ def _set_default_value(cls, prop: BaseVar):
             field.required = False
             field.default = default_value
 
+    @classmethod
```

Comment:
This will only get the default value every time, so it won't actually update with the state vars